BookingsApi/BookingsApi/Controllers/V1/HearingsController.cs

```diff
@@ -127,7 +127,7 @@ public async Task<IActionResult> AnonymiseParticipantAndCaseByHearingId(List<Gui
         [HttpGet("{groupId}/hearings")]
         [OpenApiOperation("GetHearingsByGroupId")]
         [ProducesResponseType(typeof(List<HearingDetailsResponse>), (int)HttpStatusCode.OK)]
-        [MapToApiVersion("1.0")]
+        [MapToApiVersion("2.0")]
```

Comment:
not sure what this is for - don't we want to keep v1 intact?

Reply:
thanks! I might have gotten v1 and 2 mixed up